README.md

```diff
@@ -28,6 +28,10 @@ paver test
 If you ran `bb init` and you want to do "discovery testing" with the integration tests,
 set `FORCE_DISCOVERY`environment variable to `1`.
 
+### Troubleshooting
+
+`./venv/bin/paver` - use if paver in python3 is prefered before python2 (you need them for paver)
```

Comment:
I don't understand the sentence and I can't suggest different wording, because I don't know paver 😞

Reply:
`paver` that tool in readme everywhere?

Reply:
I know what `paver` is, but I don't know what you meant. And I can't decipher it and reformulate the sentence, because I don't remember how Paver works.

Reply:
ok, refactored